lib/matplotlib/dates.py

```diff
@@ -282,6 +283,11 @@ def _from_ordinalf(x, tz=None):
         tz = _get_rc_timezone()
 
     ix = int(x)
+    if ix < 1:
+        raise ValueError('cannot convert {} < 1 to a date.  This '
```

Comment:
I think should be:
`cannot convert {} to a date`

`{} < 1` evaluate to a bool 😉

Reply:
OK.  Changed to 
```
                             'view limit minimum {} is less than 1 and '
                             'is an invalid Matplotlib date value. This '
                             'often happens if you pass a non-datetime '
                             'value to an axis that has datetime units'
                             .format(vmin))
```